### PR TITLE
Update SteamScamSite and adding the light version of it.

### PR DESCRIPTION
**Pull request by PoorPocketsMcNewHold**
No idea if i've done it correctly, or managed to still do some mistakes.

## Adding or Updating Data?

Please use the the [data model documentation](https://github.com/collinbarrett/FilterLists/wiki/Data-Model_sidebar) to verify your updates conform to project standards.

### CODE REVIEW

File: data/FilterList.json [hunk 1]
```diff
@@ -16590,4 +16591,15 @@
     "updatedDate": "2019-01-15T03:00:00",
     "viewUrl": "https://raw.githubusercontent.com/cbuijs/shallalist/master/webtv/urls"
   }
```
Comment (bogachenko) [inline data/FilterList.json]: you missed the comma and brace on line 16594

Comment (DandelionSprout) [reply data/FilterList.json]: I'm siding with Bogachenko on this one. A paragraph should normally be opened with `{` , and should be closed with `},` with a comma.

( `}` without a comma if it's the last paragraph).

File: data/FilterList.json [hunk 2]
```diff
@@ -16589,5 +16590,18 @@
     "syntaxId": 8,
     "updatedDate": "2019-01-15T03:00:00",
     "viewUrl": "https://raw.githubusercontent.com/cbuijs/shallalist/master/webtv/urls"
-  }
-]
\ No newline at end of file
+  },
+  {
+    "id": 1453,
```
Comment (collinbarrett) [inline data/FilterList.json]: Sorry, just merged #683 which used this id number. Nothing you did wrong, just a fault of the current process. Please bump this Id to the next unused number, then I'll merge. Thanks!

**Comment by DandelionSprout** [comment]
Add a new line at line 16,593 with `{`, and remove the comma after `}` in line 16,604. Hopefully that should do the trick.

**Comment by DandelionSprout** [comment]
You're getting closer to success, but now the `[` that is supposed to be at the end of all .json files is missing.

**Comment by PoorPocketsMcNewHold** [comment]
> You're getting closer to success, but now the `[` that is supposed to be at the end of all .json files is missing.

I was actually going to say that I was expecting an another character to actually close the whole list. I've known I've had to hear my presentiments. Going to add it right away.
_Hopefully, I will finally manage to understand correctly how things works._

**Comment by DandelionSprout** [comment]
I too had to learn how .json files worked, which I did by studying other existing files to learn how their syntaxes and paragraphs looked like.

And I also see that you didn't see the edit that I had to hastily make to my above comment. It wasn't meant to be `[`, but `]`. 😅

**Comment by PoorPocketsMcNewHold** [comment]
> I too had to learn how .json files worked, which I did by studying other existing files to learn how their syntaxes and paragraphs looked like.
> 
> And I also see that you didn't see the edit that I had to hastily make to my above comment. It wasn't meant to be `[`, but `]`. 😅

I didn't indeed x/
Fixing that agaaaain 😅

**Comment by DandelionSprout** [comment]
Okay, **now** I'm 95% sure that the build will succeed this time around. 👍

**Comment by collinbarrett** [comment]
Thanks, all!

**Pull request merged**